integration/v7/isolated/update_org_quota_command_test.go

```diff
@@ -29,6 +29,7 @@ var _ = Describe("update-org-quota command", func() {
 				Eventually(session).Should(Say(`-r\s+Total number of routes. -1 represents an unlimited amount.`))
 				Eventually(session).Should(Say(`--reserved-route-ports\s+Maximum number of routes that may be created with ports. -1 represents an unlimited amount.`))
 				Eventually(session).Should(Say(`-s\s+Total number of service instances. -1 represents an unlimited amount.`))
+				Eventually(session).Should(Say(`-l\s+Total log volume per second all processes can have \(e.g. 128B, 4K, 1M\). -1 represents an unlimited amount.`))
```

Comment:
@pivotalgeorge what change are you thinking?

We had thought the "e.g." section made it clear enough.

Reply:
The error section will also make it pretty clear if a user enters an incorrect value.

Reply:
Ah, the error is clearer, thanks. the wording change I would make here is: `Total log volume in bytes per second all processes can have \(e.g. 128B, 4K, 1M\)`

Reply:
> Total log volume in bytes per second all processes can have \(e.g. 128B, 4K, 1M\)

I would like to push back on this change, as I think that description may confuse users by indicating that the unit for the flag is `bytes per second`, rather than `bytes`. I'm not going to take a hard stand on this though. If you feel strongly we'll make the change :). Please let us know!

In an ideal world I think we would accept bytes per second as the unit for this flag, but given the current codebase I'm loathe to add custom code for that conversion and would vastly prefer to continue accepting bytes as the unit for this flag.

Reply:
ah that makes sense. how about `Total log volume per second all processes can have, in bytes \(e.g. 128B, 4K, 1M\)` ?

Reply:
LGTM! We'll go ahead and make that change now.